HandleNHNotificationCallActivity/handler.ts

```diff
@@ -64,7 +77,11 @@ export const getCallNHServiceActivityHandler = (
             retryActivity(context, `${logPrefix}|ERROR=${toString(e)}`)
           );
         case NotifyKind.Notify:
-          return notify(message.installationId, message.payload)
+          return notify(
+            nhService.value,
+            message.installationId,
+            message.payload
+          )
             .mapLeft(e =>
               retryActivity(context, `${logPrefix}|ERROR=${toString(e)}`)
             )
```

Comment:
Those `mapLeft` are the same and can be moved outside the chain

Reply:
This is what I copied from io-functions-app, but since we are going to split them, do you think it's necessary too do this refactoring?

Reply:
Not necessary